.azure-pipelines/default.yml

```diff
@@ -0,0 +1,40 @@
+trigger:
+- master
+
+jobs:
+- job: 'Windows'
+
+  pool:
+    vmImage: 'vs2017-win2016'  
+
+  steps:
+  - script: |
+     python -m venv .\
+     
+     pip install git+https://github.com/lmazuel/swagger-to-sdk
+     pip install -e "git+https://github.com/Azure/azure-sdk-for-python#subdirectory=azure-sdk-tools&egg=azure-sdk-tools"
+
+    displayName: 'Prep Environment, Install Packages copy'
+
+  - powershell: |
+     $inputFilterSet="$(build_targeting_string)"
```

Comment:
nit: If you decided to keep the default value in the yml instead of the build definition I think I would change this code to look like `$inputFilterSet = if ("$(build_targeting_string)" -ne "") { "$(build_targeting_string)" } else { "azure-*" }`. That make it clear you are just trying to initialize that variable and make the code more compact.